environment.rb

```diff
@@ -0,0 +1,97 @@
+module Environment
+    class Depot
+      attr_accessor :packs 
+      
+      def initialize
+        @packs = {
+          simple_transportation_pack: { 
+            intelligence:[:cellphone],
+            tems: [:medipack, :chevy_versa]
+          },
+          standard_transportation_pack: {
+            intelligence: [:cellphone, :antena],
+            arsenal: [:colt_1911],
+            items: [:handcuffs, :medipack, :chemistry, :chevy_versa]
+          },
+          simple_mission_pack: {
+            intelligence: [:infopack, :laptop, :cellphone, :antena],
+            arsenal: [:colt_1911],
+            items: [:handcuffs, :medipack,:chemistry,:finalcial]
+          },
+          standard_mission_pack: {
+            intelligence: [:infopack, :laptop, :cellphone, :antena],
+            arsenal: [:remington_870, :colt_1911, :machete, :hatchet],
+            items: [:handcuffs, :medipack, :chemistry, :finalcial]
+          } 
+        }
+       end
+      end
+  
+    class Control
+      attr_accessor :missions
+      
+      def initialize
+        @missions = {}
+      end
+      
+      status = %w(paused aborted failed accomplished)
+  
+      status.each do |s|
+        define_method("set_mission_to_#{s}") do |args|
+          if !(@missions[args].nil?)
+            @missions[args][:status] = s.to_sym
+          end
+        end
+      end
+      
+      def new_mission(name: , objective: , pack: )
+        @missions[name] = {objective: objective, status: :active, pack: pack }
```

Comment:
Little detail, I am sure you know it, leave an space at the beginning of the hash. `{ objective....`